legal-api/src/legal_api/reports/report.py

```diff
@@ -869,19 +869,18 @@ def _format_change_of_registration_data(
         if business_office := filing.get(filing_type).get("offices", {}).get("businessOffice"):
             filing["offices"] = {}
             filing["offices"]["businessOffice"] = business_office
-            offices_json = VersionedBusinessDetailsService.get_office_revision(
-                prev_completed_filing, self._filing.legal_entity_id
-            )
-            filing["offices"]["businessOffice"]["mailingAddress"]["changed"] = self._compare_address(
-                business_office.get("mailingAddress"), offices_json["businessOffice"]["mailingAddress"]
-            )
-            filing["offices"]["businessOffice"]["deliveryAddress"]["changed"] = self._compare_address(
-                business_office.get("deliveryAddress"), offices_json["businessOffice"]["deliveryAddress"]
-            )
-            filing["offices"]["businessOffice"]["changed"] = (
-                filing["offices"]["businessOffice"]["mailingAddress"]["changed"]
-                or filing["offices"]["businessOffice"]["deliveryAddress"]["changed"]
-            )
+            offices_json = VersionedBusinessDetailsService.get_office_revision(prev_completed_filing, self._business)
+            if offices_json:
```

Comment:
this is the same situation as https://github.com/bcgov/lear/blob/1b40e6d07ba8b1d090b4ffd21ca3aaef102f1533/legal-api/src/legal_api/services/business_details_version.py#L216-L219 I believe.

Reply:
Updated